lib/os_mon/doc/src/disksup.xml

```diff
@@ -57,15 +57,40 @@
       longer valid.</p>
   </description>
 
+  <datatypes>
+    <datatype>
+      <name>time()</name>
+      <desc>
+        <p>Supported units:</p>
+        <taglist>
+          <tag><c>int()</c></tag>
+          <item>
+            <p>The time interval in minutes.</p>
+          </item>
+          <tag><c>{second, int() >= 1}</c></tag>
+          <item>
+            <p>The time interval in seconds.</p>
+          </item>
+          <tag><c>{millisecond, int() >= 1}</c></tag>
+          <item>
+            <p>The time interval in milliseconds.</p>
+          </item>
```

Comment:
```suggestion
          <tag><c>{TimeUnit, Time}</c></tag>
          <item>
            <p>
              The time interval <c>Time</c> in a time unit specified by <c>TimeUnit</c> where
              <c>TimeUnit</c> is of the type
              <seetype marker="erts:erlang#time_unit/0"><c>erlang:time_unit/0</c></seetype>
              and <c>Time</c> is a positive integer. The time interval needs to be at least one
              millisecond long.
            </p>
          </item>
```
I see no reason to limit time unit to `second` and `millisecond`. I think it is much better to use the already defined time unit type.

Note that I've not tested building the docs using the above so the doc build might complain about it.

Reply:
This is better. I was initially worried of silently `floor()`ing values with parts of a millisecond but I see now that it just needed better documentation.